app/person-escort-record/controllers/framework-overview.js

```diff
@@ -2,7 +2,7 @@ const presenters = require('../../../common/presenters')
 
 function frameworkOverview(req, res) {
   const { originalUrl, framework, personEscortRecord = {}, move } = req
-  const profile = personEscortRecord?.profile || move?.profile
+  const profile = move?.profile || personEscortRecord?.profile
```

Comment:
Really struggling to understand how the move could lack a profile but a PER exists...

Reply:
Sorry, got trigger happy on the merge! It was actually around when the person escort record app was included within the move app. 

So a PER will exist on the profile, but we haven't asked for the profile to be included as part of the request. We also aren't running the `transform` method on the profile when it comes back from the move so the `fullname` property wasn't available. Inverting this statement meant it would continue to work in both instances of the app being loaded.

Ideally, the best fix would be to utilise some sort of transformation level at devour level when each model is deserialised but unfortunately the library doesn't support that as easily as we would like.